magpie/setup/magpie-setup-project-zeppelin

```diff
@@ -239,19 +266,7 @@ post_shiroini=${ZEPPELIN_CONF_DIR}/shiro.ini
 
 if [ "${SPARK_SETUP_TYPE}" == "YARN" ];
 then
-    # sets magpie_sparkmajorminorversion
-    Magpie_get_spark_major_minor_version ${SPARK_VERSION}
-
-    # Handle special settings depending on version
-    # 0 is =, 1 is >, 2 is <
-    Magpie_vercomp ${magpie_sparkmajorminorversion} 2.0
-    vercomp_result=$?
-    if [ "${vercomp_result}" == "0" ] || [ "${vercomp_result}" == "1" ]
-    then
-        zeppelin_spark_master="yarn"
-    else
-        zeppelin_spark_master="yarn-client"
-    fi
+    zeppelin_spark_master="yarn-client"
```

Comment:
why does this no longer need to be set to "yarn" for older versions?  I would think this part shouldn't change?

Reply:
oh is it b/c you assume a different default spark of > 2.0?  I think we should still assume the user may choose an older spark.

Reply:
I think the statement was stating that you should use yarn when the spark version is greater than or equal to 2.0. I tested spark with both 2.0, and 2.3 using yarn-client and they both were fine. I think this was in here by mistake at one point as zeppelin seems fine with yarn-client for all versions of spark.

Reply:
ohh, i misread the code.  My bad.